test/testdata

```diff
@@ -199,37 +199,38 @@ Scan      ORCA         OrcaScanTest                   basicORCA3.0          dvb_
 Scan      Jaguar       GenericScanTest_optdone_bool   basicJaguar8.3        dvb_scan.out
 Scan      Jaguar       JaguarScanTest                 basicJaguar8.3        dvb_scan.out
 
-SP        ADF         ADFSPTest             basicADF2007.01       dvb_sp_b.adfout
-SP        ADF         ADFSPTest             basicADF2013.01       dvb_sp_b.adfout
-SP        DALTON      GenericSPTest         basicDALTON-2013      dvb_sp_hf.out
-SP        DALTON      GenericSPTest         basicDALTON-2013      dvb_sp_ks.out
-SP        DALTON      GenericSPTest         basicDALTON-2015      dvb_sp_hf.out
-SP        DALTON      GenericSPTest         basicDALTON-2015      dvb_sp_ks.out
-SP        GAMESSUK    GenericSPTest         basicGAMESS-UK7.0     dvb_sp_b.out
-SP        GAMESSUK    GenericSPTest         basicGAMESS-UK8.0     dvb_sp_hf.out
-SP        GAMESSUK    GenericSPTest         basicGAMESS-UK8.0     dvb_sp_ks.out
-SP        GAMESS      GenericSPTest         basicFirefly8.0       dvb_sp.out
-SP        GAMESS      GenericSPTest         basicGAMESS-US2014    dvb_sp.out
-SP        GAMESS      GenericSPTest         basicGAMESS-US2017    dvb_sp.out
-SP        Gaussian    GenericSPTest         basicGaussian03       dvb_sp.out
-SP        Gaussian    GenericSPTest         basicGaussian09       dvb_sp.out
-SP        Gaussian    GenericSPTest         basicGaussian16       dvb_sp.out
-SP        Jaguar      Jaguar7SPTest         basicJaguar7.0        dvb_sp.out
-SP        Jaguar      GenericSPTest         basicJaguar8.3        dvb_sp_hf.out
-SP        Jaguar      GenericSPTest         basicJaguar8.3        dvb_sp_ks.out
-SP        Molcas      GenericSPTest         basicOpenMolcas18.0   dvb_sp.out
-SP        Molpro      GenericSPTest         basicMolpro2006       dvb_sphf.out
-SP        Molpro      GenericSPTest         basicMolpro2012       dvb_sphf.out
-SP        NWChem      GenericSPTest         basicNWChem6.0        dvb_sp_hf.out
-SP        NWChem      GenericSPTest         basicNWChem6.0        dvb_sp_ks.out
-SP        NWChem      GenericSPTest         basicNWChem6.5        dvb_sp_hf.out
-SP        NWChem      GenericSPTest         basicNWChem6.5        dvb_sp_ks.out
-SP        ORCA        OrcaSPTest            basicORCA2.9          dvb_sp.out
-SP        ORCA        OrcaSPTest            basicORCA3.0          dvb_sp.out
-SP        Psi         Psi3SPTest            basicPsi3             dvb_sp_hf.out
-SP        Psi         GenericSPTest         basicPsi4.0           dvb_sp_rhf.out
-SP        Psi         GenericSPTest         basicPsi4.0           dvb_sp_rks.out
-SP        QChem       GenericSPTest         basicQChem4.2         dvb_sp.out
+SP        ADF         ADFSPTest             basicADF2007.01             dvb_sp_b.adfout
```

Comment:
Why are all these lines changed?

Reply:
Just changed the formatting so that the Turbomole files are more visible and readable.

Reply:
I still see the lines in this block as changed.

Reply:
Yeah. I changed the spacing in between the Program name and the test file name. I did this because the there wasn't much space in between `basicTurbomole/dvb_sp` and `basis` and it was hard to read.

Reply:
Ack